brainscore/benchmarks/hebart2023.py

```diff
@@ -14,39 +18,32 @@
           }"""
 
 class Hebart2023Accuracy(BenchmarkBase):
-    def __init__(self):
+    def __init__(self, similarity_measure):
         self._metric = None
-        self._assembly = None
         self._visual_degrees = None
-
         self._number_of_trials = None
-        
-        #TODO I am not sure where this goes , but it should be in the benchmark right?
-        self.validation_data = None  # https://osf.io/b2a4j we'll need to convert this to BehavioralAssembly right?
-
-        # TODO do we put the similarity matrix inside look_at?
-        #self.validation_data_pairs = [[self.validation_data[:, 0], self.validation_data[:, 1]], # All pairs  
-        #                        [self.validation_data[:, 0], self.validation_data[:, 2]], # TODO change to BehavioralAssembly
-        #                        [self.validation_data[:, 1], self.validation_data[:, 2]]]
+        self.validation_data = None  # https://osf.io/b2a4j -> host on S3 
 
+        #  What is the difference here? What should I use?
+        self._assembly = LazyLoad(lambda: load_assembly("Hebart2023"))     # <- this is from Geirhos er al.
+        def load_assembly(dataset):                                        # 
+            pass                                                           #
+        self.stimulus_set  = brainscore.get_stimulus_set("Hebart2023")     # <- this is from Islam2021
+        
         super(Hebart2023Accuracy, self).__init__(
-            identifier=None, version=1,
-            ceiling_func=None,
-            parent=None,
+            identifier=f'Hebart2023Accuracy_{similarity_measure}', version=1,
+            ceiling_func=None,                                             # Will need the noise ceiling from Martin H.
```

Comment:
ideally we can compute this directly from the data to make the ceiling estimate reproducible